### PR TITLE
fasttext: init at 0.2.0

### PR DESCRIPTION
**Pull request by danieldk**
###### Motivation for this change

This change adds a derivation for fasttext. Fasttext is a tool + library for training word embeddings with subword units and text classification models. Fasttext is a popular tool in the computational linguistics and natural language processing communities.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by danieldk** [comment]
@grahamcofborg build fasttext

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -14051,6 +14051,8 @@ in
 
   lwan = callPackage ../servers/http/lwan { };
 
+  fasttext = callPackage ../applications/science/machine-learning/fasttext { };
```
Comment (c0bw3b) [inline pkgs/top-level/all-packages.nix]: It should rather be in the `### APPLICATIONS` section, right after `exrtools`
(and yes `labelImg` is also misplaced...)

Comment (danieldk) [reply pkgs/top-level/all-packages.nix]: Thanks! Fixed.

**Comment by c0bw3b** [comment]
To check on Darwin:
@GrahamcOfBorg build fasttext

**Review by c0bw3b — APPROVED** [review]

**Pull request merged**